cpp/src/prims/kv_store.cuh

```diff
@@ -172,7 +172,7 @@ struct kv_binary_search_store_device_view_t {
   {
   }
 
-  __device__ value_type find(key_type key) const
+  __host__ __device__ value_type find(key_type key) const
```

Comment:
Why did you add __host__ here? I don't think this function is host callable, and did you add this to fix some build error? There might be a better way than marking this only device callable function to host device.